Python/Scrap_Latest_Torrents/scrap_latest_torrents.py

```diff
@@ -0,0 +1,68 @@
+"""
+This script downloads the torrents in our Torrent client from the
+website https://rarbg.to/ from an entered label.
+"""
+
+import subprocess
+import os
+import sys
+import rarbgapi
+
+
+def main():
+    """
+    Principal function
+    """
+    count = 0
+
+    # Label to search
+    label = input("Enter a label: ")
+
+    print("Searching torrents... please wait")
+
+    # API RARGB
+    client = rarbgapi.RarbgAPI()
```

Comment:
You can abstract the API call into another UDF. So it becomes more readable and flexbile.

Reply:
I add a new function